install.sh

```diff
@@ -23,28 +23,32 @@ docker exec magento2-container bin/magento setup:di:compile
 # Clear cache
 docker exec magento2-container bin/magento cache:flush
 
-# Install Express Checkout
-docker exec magento2-container composer require adyen/adyen-magento2-expresscheckout
-docker exec magento2-container bin/magento module:enable Adyen_ExpressCheckout
-
```

Comment:
This block should exist in the installation script for express module installation.